playground/Makefile

```
@@ -0,0 +1,18 @@
+.PHONY: build
+build:
+	podman build -f playground/Containerfile -t ghcr.io/ai-lab-recipes/playground --format docker playground
+
+models/llama-2-7b-chat.Q5_K_S.gguf:
```

```suggestion
.PHONEY: download
download:
```
does it make sense to make this "download" instead of the model name?